init.zsh

```diff
@@ -4,11 +4,11 @@ typeset -gA FZF_WIDGETS_OPTS
 : "Create cache directory" && () {
   if [[ -n $XDG_CACHE_HOME ]]; then
     [[ ! -d $XDG_CACHE_HOME ]] && mkdir $XDG_CACHE_HOME
-    local dir="$XDG_CACHE_HOME/fzf-widgets"
+    local dir="$XDG_CACHE_HOME/fzf-widgets-$USER"
```

Comment:
Please remove `-$USER` because $XDG_CACHE_HOME is user-specific directory. (ex. `$HOME/.cache`)